crates/wallets/Cargo.toml

```diff
@@ -27,15 +27,9 @@ alloy-dyn-abi.workspace = true
 
 # aws-kms
 alloy-signer-aws = { workspace = true, features = ["eip712"], optional = true }
-aws-config = { workspace = true, default-features = true, optional = true }
```

Comment:
we need to keep this line because we need default-features = true. you will also need to add `#[cfg(feature = "aws-kms"] use aws_config as _;`